docs/source/tutorials/image.ipynb

```diff
@@ -291,25 +626,32 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "predicted_labels = pred_probs.argmax(axis=1)\n",
-    "acc = accuracy_score(labels, predicted_labels)\n",
-    "print(f\"Cross-validated estimate of accuracy on held-out data: {acc}\")"
+    "lab = Datalab(data=dataset, label_name=\"label\", image_key=\"image\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 6. Use cleanlab to find label issues\n"
+    "The `find_issues` method can automatically infer the types of issues to be checked for based on the provided arguments. Here, we provide `features` and `pred_probs` as arguments. If you want to check for a specific issue type, you can do so using the `issue_types` argument. Check the [documentation](https://docs.cleanlab.ai/stable/cleanlab/datalab/datalab.html#cleanlab.datalab.datalab.Datalab.find_issues) for more comprehensive guide on `find_issues` method."
```

Comment:
this is not the right way to format links in tutorials, look at DEVELOPMENT.md or ask @elisno